charts/plex-media-server/README.md

```diff
@@ -0,0 +1,133 @@
+# plex-media-server Chart
+===========
+
+A Helm chart for deploying the a Plex Personal Media Server(PMS) server.
+
+This is not an officially supported Plex product. This project is actively being maintained.
+
+
+### Sample init Container scripts
+
+If you already have a different PMS server running elsewhere and wish to migrate it to be running in Kubernetes
+the easiest way to do that is to import the existing PMS database through the use of a custom init script.
+
+**Note: the init script must include a mechanism to exit early if the pms database already exists to prevent from overwriting its contents**
+
+The following script is an example (using the default `alpine` init container) that will pull
+a tar gziped file that contains the pms `Library` directory from some web server.
+
+```sh
+#!/bin/sh
+echo "fetching pre-existing pms database to import..."
+
+if [ -d "/config/Library" ]; then
+  echo "PMS library already exists, exiting."
+  exit 0
+fi
+
+apk --update add curl
```

Comment:
nit containerized Alpine does not need to [update locks](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache):

```suggestion
apk --no-cache add curl
```